### PR TITLE
Display image copyright in HTML element

### LINKED ISSUE #456: Image Copyright
**Is your feature request related to a problem? Please describe.**
Ensure the copyright holder of non-BBC images is displayed and read correctly by screen readers.

**Describe the solution you'd like**
- Overlay the copyright using text (as opposed to having ImageChef add the copyright as a watermark).
- The visible copyright holder should have a contrast of 4.5:1

### PR DESCRIPTION
**Pull request by bcmn**
Resolves #456 

Displays image copyright as a HTML element, overlaid on the bottom-right of the image. At some point we expect this to be unnecessary as it will be burnt into the image, but currently we have no indication as to how soon that might be the case.

* Creates copyright component with unit test
* Ensures both visual & AT text should read "© <Copyright Holder>", e.g. "© Getty Images"
* Creates Minion typography group
* Creates ImageWrapper with unit test, required to ensure correct placement of the copyright element with regards to both the image & the caption.
* Changes fixture data image, to one more typical -- previous had embedded source & logo. [Previous](https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png) | [New](https://ichef.bbci.co.uk/news/640/cpsprodpb/E7DB/production/_101655395_paulineclayton.jpg)
* Changes fixture data copyright value, as it is no longer visually hidden text.
* Updates fixture to include ImageWrapper

[Relevant Storybook page](http://localhost:9001/?selectedKind=Figure&selectedStory=with%20non-BBC%20copyright&full=1&addons=1&stories=1&panelRight=0).

Note also [this accessibility note from Catharine](https://github.com/bbc/simorgh/issues/456#issuecomment-418804230).

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by bcmn** [comment]
Code climate is complaining about something that was not added in this ticket, @pjlee11 is fixing separately.

File: src/app/components/Figure/Copyright/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+// const copyrightSymbolPrefix = '&#169; ';
+const copyrightSymbolPrefix = '© ';
```
Comment (pjlee11) [inline src/app/components/Figure/Copyright/index.jsx:13]: Curious on other reviewers opinion of if we should use the encoded version of this symbol or keep as is

File: src/app/components/Figure/fixtureData.jsx [hunk 2]
```diff
@@ -2,39 +2,48 @@ import React from 'react';
...
+      <Image alt={imageAlt} src={imageSrc} />
+    </ImageWrapper>
     <Caption>{captionValue}</Caption>
   </Figure>
 );
 
 export const FigureImageWithCopyright = (
   <Figure>
-    <Image alt={imageAlt} src={imageSrc} />
-    <VisuallyHiddenText>{copyrightText}</VisuallyHiddenText>
```
Comment (pjlee11) [inline src/app/components/Figure/fixtureData.jsx:30]: The code climate warnings mentioned here are being addressed in https://github.com/bbc/simorgh/pull/567

File: src/app/components/Figure/Copyright/index.jsx (hunk 1 above)
Comment (DemonusPC) [reply src/app/components/Figure/Copyright/index.jsx:13]: How sure are we that using the encoded one will work when it is sent to the client?

**Review by jtart — CHANGES_REQUESTED** [review]
@pjlee11 @bcmn this PR is quite large. Can you please split it into changes to individual React components and incremental integrations of those changes?

i.e. create the Copyright component in a PR, then integrate that Copyright component in separate PR.

**Pull request closed without merging**